### DISCUSSION TITLE
带密码的文章不显示目录&

### DISCUSSION THREAD

**Discussion by hanrongMan** [tangly1024/NotionNext#979] [Q&A]
1.给文档设置了密码后，输入密码点进去后，不能显示文档的目录,例如下图的目录：
<img width="303" alt="image" src="https://user-images.githubusercontent.com/19281198/232296131-8585e57e-4c8f-4684-8d51-7823fbfd4e98.png">

2. 如果文档整体是放在sync block 中时，位于头部的目录显示就会是一个黑条,像这样
<img width="837" alt="image" src="https://user-images.githubusercontent.com/19281198/232296735-cc7a219c-b771-44f9-ab98-b20049c4dead.png">

**Comment by hanrongMan** [comment]
@txs @tangly1024

**Reply by luoliguang** [reply]
@tangly1024
